core/src/main/java/nl/nn/adapterframework/jdbc/dbms/GenericDbmsSupport.java

```diff
@@ -387,4 +389,62 @@ public String getBooleanFieldType() {
 	public String getBooleanValue(boolean value) {
 		return (""+value).toUpperCase();
 	}
+
+	@Override
+	public String convertQuery(Connection conn, String query, String dbmsFrom) throws SQLException, JdbcException {
+		if (isQueryConvertable(dbmsFrom)) {
+			warnConvertQuery(dbmsFrom);
+		}
+		return query;
+	}
+	
+	protected void warnConvertQuery(String dbmsFrom) {
+		log.warn("don't know how to convert queries from [" + dbmsFrom + "] to [" + getDbmsName() + "]");
+	}
+
+	protected boolean isQueryConvertable(String dbmsFrom) {
```

Comment:
Zou hier ook  steeds 'fromSqlDialect' in plaats van 'dbmsFrom' gebruiken

Reply:
https://github.com/ibissource/iaf/pull/251/commits/94587d8c268f617ced098054c3f9e74e58d9c4e0